File: exercicios-java/estrutura-sequencial/exe04.java
import java.util.Scanner;

/*
Faça um Programa que peça as 4 notas bimestrais e mostre a média.
*/

public class exe4{

  public static void main(String[] args){
    Scanner leitor = new Scanner(System.in);

    float n1=0, n2=0, n3=0, n4=0, media=0;


    // nextInt() 
    // nextFloat()
    // nextDouble()
    System.out.print("Informe a primeira nota: ");
    n1 = leitor.nextFloat();
    System.out.print("Informe a segunda nota: ");
    n2 = leitor.nextFloat();
    System.out.print("Informe a terceira nota: ");
    n3 = leitor.nextFloat();
    System.out.print("Informe a quarta nota: ");
    n4 = leitor.nextFloat();

    media = (n1+n2+n3+n4) / 4;

    System.out.printf("A média é %.2f", media);


  }
}

File: exercicios-javascript/estrutura-sequencial/exe04.js

/*
Faça um Programa que peça as 4 notas bimestrais e mostre a média.
*/

const readline = require('readline-sync')

let nota1 = readline.questionFloat('Informe a primeira nota: ')
let nota2 = readline.questionFloat('Informe a segunda nota: ')
let nota3 = readline.questionFloat('Informe a terceira nota: ')
let nota4 = readline.questionFloat('Informe a quarta nota: ')

let media = (nota1 + nota2 + nota3 + nota4) / 4

/*
existem varios metodos para formatar a saida de números
o que eu irei usar é o

 toFixed()

*/
console.log();
console.log(`A media do aluno eh ${media}`) // sem formatação
console.log(`A media do aluno eh ${media.toFixed(2)}`) // com toFixed()
